media/js/mozorg/home/home-a.js

```diff
@@ -115,6 +116,25 @@
             handler: navState('firefox', 'innovation'),
             offset: '30%'
         });
+
+        surveyWaypoint = new Waypoint({
+            element: '#firefox',
+            handler: function(direction) {
+                var $surveyMsg = $('#survey-message');
+                if ($surveyMsg.length) {
+                    if (direction === 'down') {
+                        // slide up when scrolling down
+                        $surveyMsg.addClass('stuck').css({ bottom: '-' + $surveyMsg.height() + 'px' }).animate({ bottom: '0' }, 500);
```

Comment:
Instead of asking the browser to calculate the height of the element (which is expensive), I'd recommend just picking a large enough static value and sticking to that. Looks like the element is about 38px tall, so if you just say the offset is like 90px, you'd be pretty safe at both one and two lines of text.